.gitignore

```diff
@@ -9,7 +9,6 @@ tmp
 
 # dependencies
 node_modules
-package-lock.json
```

Comment:
Why do we want to ignore the package.lock?

Reply:
We want to not ignore it anymore. It's so in the future we can use any changes to the package-lock.json file to identify if we need to install npm dependencies again or cache them to speed up builds.

Reply:
Sorry! I looked at this quickly and thought you were adding it the ignore, which I could not for the life of me understand why. Please disregard, its been a long day!

Reply:
Totally understand. I've been there.